sdk/spring/azure-spring-boot-test-core/src/main/java/com/azure/spring/test/EnvironmentVariable.java

```diff
@@ -48,6 +49,7 @@
     public static final String SPRING_TENANT_ID = System.getenv("SPRING_TENANT_ID");
     public static final String SPRING_JMS_STANDARD_SERVICEBUS_CONNECTION_STRING = System.getenv("SPRING_JMS_STANDARD_SERVICEBUS_CONNECTION_STRING");
     public static final String SPRING_JMS_PREMIUM_SERVICEBUS_CONNECTION_STRING = System.getenv("SPRING_JMS_PREMIUM_SERVICEBUS_CONNECTION_STRING");
+    public static final String APPLICATION_SERVER_PORT = System.getenv("APPLICATION_SERVER_PORT");
```

Comment:
Can we use fixed ports?
Like:
1. webapp: 8880.
2. webapiA: 8881
3. webapiB: 8882.